templates/jaasai/experts/omnivector.html

```diff
@@ -0,0 +1,171 @@
+      </div>
+      <ul>
+        <li class="p-list__item">2 Node; Spark + Jupyter Notebook.</li>
+        <li class="p-list__item">Integrates with Omnivector&apos;s custom storage solutions.</li>
+        <li class="p-list__item">Scale this bundle up to 1000s of nodes!</li>
+      </ul>
+      <a href="{{ external_urls.gui }}?dd=~omnivector/bundle/spark-jupyter-core" class="p-button--positive">Deploy Spark Cluster</a>
+
+    </div>
+    <div class="col-6">
+      <h3><a href="{{ url_for('jaasstore.user_entity', username='omnivector', charm_or_bundle_name='slurm-core') }}">
+        SLURM Core Bundle</a>
+      </h3>
+      <p>Ominvector’s SLURM Core bundle provides the key components of the SLURM workload manager including 1x
+      slurm-controller, 1 x slurm-node, 1 x slurm-dbd.</p>
+      <div class="pictogram-bg">
+        <object data="{{ external_urls.charmstore }}~omnivector/bundle/slurm-core/diagram.svg" type="image/svg+xml"
+          style="min-height:280px" title="SLURM Core Bundle overview"></object>
+      </div>
+      <ul>
+        <li class="p-list__item">4 Node SLURM starter bundle.</li>
+        <li class="p-list__item">Get up and running by deploying this bundle on your local machine, then deploy to your
+          favourite cloud when you are ready to take your solution live.</li>
+        <li class="p-list__item">SLURM packaging experts.</li>
+      </ul>
+      <a href="{{ external_urls.gui }}?dd=~omnivector/bundle/slurm-core" class="p-button--positive">Deploy SLURM Core</a>
+    </div>
+  </div>
+  <div class="row">
+    <div class="col-6">
+      <h3>
+        <a href="{{ url_for('jaasstore.user_entity', username='omnivector', charm_or_bundle_name='spark-jupyter-conda-extended') }}">
+          Big-data processing with CEPH and real-time monitoring.
+        </a>
+      </h3>
+      <p>Ominvector’s Spark Extended bundle provides a complete data processing solution at the push of a button.</p>
+      <div class="pictogram-bg u-hide--small">
+        <object data="{{ external_urls.charmstore }}~omnivector/bundle/spark-jupyter-conda-extended-0/diagram.svg"
+          type="image/svg+xml" style="min-height:280px" title="Spark Core overview"></object>
+      </div>
+      <ul>
+        <li class="p-list__item">Enhanced Spark Operations</li>
+        <li class="p-list__item">Integrated CEPH Storage</li>
+        <li class="p-list__item">Enterprise Grade Monitoring</li>
+      </ul>
+      <a href="{{ url_for('jaasstore.user_entity', username='omnivector', charm_or_bundle_name='spark-jupyter-conda-extended') }}" class="p-button--positive">
+        Deploy Spark Cluster with Monitoring
+      </a>
+    </div>
+    <div class="col-6">
+      <h3>
+        <a href="{{ url_for('jaasstore.user_entity', username='omnivector', charm_or_bundle_name='slurm-openfoam') }}">
+          Custom HPC Solutions</a>
```

Comment:
Can we align the svgs ? 
Also the openfoam diagram doesn't exist.

<img width="1304" alt="Screen Shot 2019-09-04 at 9 37 10 AM" src="https://user-images.githubusercontent.com/532033/64269722-a448f700-cef7-11e9-9849-7aa1e5283503.png">

Reply:
Seeing the same, missing bundle

Reply:
Fixed the bundle.

As for aligning SVGs - easiest way is to trim the left-hand title - @clagom ?